design/milestone_1_2_design_doc.md

```diff
@@ -366,8 +366,9 @@ Low level design of the solution will be introduced in a separate document.
 
 #### Customer experience
 
-This solution does not introduce any additional requirements for the solution besides using `Deployment` instead of a `Job`.
-We might add some variables, all with reasonable defaults as to ensure no additinal steps are needed to migrate from Milestone 1 to 2.
+This solution introduce only 1 additional requirement for the customer; Use updated Conjur that includes the new API.
```

Comment:
introduces*

Reply:
.  They must updated their Conjur version to access the new API

Reply:
Nit all requirements are mandatory so maybe use another word?

> This requirement is not mandatory; If this requirement is not fulfilled, the solution will fallback to option (1.a), with efficiency price. The Secrets Provider chart will be using Deployment instead of a Job. 

"
This is not mandatory. If the requirement if not fulfilled, the solution will fall back to option **1.a** but at the price of efficiency.
<ADD new line>
The Secrets Provider Chart
 "